tests/test-data/ialirt_ec2/Dockerfile

```diff
@@ -1,17 +1,3 @@
-# This code is used to Dockerize test_app.py. The workflow is as follows:
```

Comment:
Did you mean to include this? It's not strictly part of a docs update. I'm fine leaving this change in this PR because it's minor, just wanted to make sure it was intentional